### DISCUSSION TITLE
Have cooldown weeks or days for breaking changes in CI

### DISCUSSION THREAD

**Discussion by gustavovalverde** [ZcashFoundation/zebra#3927] [Q&A]
Sometimes making a bunch of changes to CI might have a great impact on development if something fails. We might want to reduce this changes in specific dates.

**Comment by dconnolly** [comment]
Proposal: for our two-week sprints, the second week/week leading up to a release/tag, we pause merging changes to CI until the tag is done. Fixes to CI would still be allowed to merge

**Comment by oxarbitrage** [comment]
Proposal: freeze the CI changes in the second way of the sprint and create the release in that week.

**Comment by teor2345** [comment]
Extra Proposal: When the PR backlog is large or urgent, we pause breaking CI changes until those PRs merge.

Here are some rough guidelines for "large or urgent":
* more than 20 PRs
* any critical PRs
* more than one high-priority PR

**Comment by mpguerra** [comment]
I feel like we already do this now
